wger/core/templates/user/login.html

```diff
@@ -5,17 +5,43 @@
 
 
 {% block header %}
+<link rel="stylesheet" type="text/css" href="{% static 'css/social_login.css' %}">
+
 {% endblock %}
 
 {% block content %}
+
+<section>
 <form action="{% url 'core:user:login' %}{% if next %}?next={{next}}{% endif %}"
       method="post"
       class="form-horizontal">
     {% trans 'Login' as submit_text %}
     {% render_form_fields form submit_text %}
-</form>
-{% endblock %}
 
+    <br>
+    <p><strong>-- OR --</strong></p>
+
+
+    <div class="col-sm-6 form-group">
+        <a href="{% url 'social:begin' 'twitter' %}" class="twitter btn btn-block" role="button">
+            <i class="fa fa-twitter fa-fw"></i>
+            Login with Twitter
+        </a>
+
+        <a href="{% url 'social:begin' 'facebook' %}" class="facebook btn btn-block" role="button">
+            <i class="fa fa-facebook fa-fw"></i>
+            Login with Facebook
+        </a>
+        <a href="{% url 'social:begin' 'google-oauth2' %}" class="google btn btn-block" role="button">
+            <i class="fa fa-google fa-fw"></i>
+            Login with Google
+        </a>
+              </div>     
+              </form>
+           
+      </section>
+      
+{% endblock %}
 
```

Comment:
I think this is not the right location to include this section. Given the app extends a lot of HTML templates. Am saying this just because of the overlap that comes in when the device is somehow minimized. Consider changing this section to base.html since its the same location where the existent login form was placed.
![image](https://user-images.githubusercontent.com/22131999/59092286-dc04ca80-8919-11e9-8246-94af90e0e03a.png)

Reply:
@reifred am working on this, will be updating you soon.

Reply:
@reifred This has been resolved. 
<img width="498" alt="Screenshot 2019-06-07 at 13 52 30" src="https://user-images.githubusercontent.com/9926422/59099375-92bd7680-892b-11e9-8584-972b5a2bf992.png">